.env

```diff
@@ -22,7 +23,10 @@ STAGE_WEB_GUEST_PORT=80
 STAGE_WEB_HOST_PORT=80
 STAGE_WEB_GUEST_SECURE_PORT=443
 STAGE_WEB_HOST_SECURE_PORT=443
+IS_STAGING=1
 
+PROD_CERTBOT_EMAIL="admin@devopsloft.io"
```

Comment:
this email box doesn't exists yet.  Is it required?

Reply:
the email is for https://letsencrypt.org/ (that sign on the certificate) 
in order to send a notification when the certificate date is expired 

the certificate shouldn't be expired because there is script that renew the certificate if needed

Reply:
maybe it is a good thing. An email in code might be used by bots for spam.